test/test-util.js

```diff
@@ -10,9 +10,33 @@ describe('Unlimited Token', function () {
 
         const factory = await createTokenFactory();
         token = await unlimitedSupplyTokenContract(
-            factory.deployUnlimitedSupplyToken('Test Token', 'TST', await owner.getAddress()),
+            factory.deployUnlimitedSupplyToken('Test Token', 'TST', [], await owner.getAddress()),
         );
     });
+
+    it('Cannot mint when not in minter list', async () => {
+        try {
+            await token.transfer(await receiver.getAddress(), parseEther('1000'));
```

Comment:
@GarfDev You should be able to test for revert messages too! Something like this:
```javascript
expect(token.transfer(await receiver.getAddress(), parseEther('1000'))).to.be.revertedWith('ERC20: transfer amount exceeds balance');
```